js/index.js

```diff
@@ -40,3 +40,59 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+let headerImg = document.getElementById("cta-img");
+headerImg.setAttribute('src', siteContent.cta["img-src"]);
+
+let midImg = document.getElementById("middle-img");
+midImg.src = siteContent["main-content"]["middle-img-src"];
+
+let navItems = document.querySelectorAll('header nav a');
+navItems.forEach((item, index) => {
+  let tempKey =`nav-item-${index + 1}`;
+  item.textContent = siteContent.nav[tempKey];      // From Brandon, good idea
+  item.style.color = "green";
+})
+
+
+let h1Text = document.querySelector('h1');
+h1Text.innerHTML = "DOM <br> Is <br> Awesome";
+
+let btnTxt = document.querySelector('button');
+btnTxt.textContent = siteContent.cta["button"];
+
+let h4Arr = document.querySelectorAll('.text-content h4');
+h4Arr[0].textContent = "Features";
+h4Arr[1].textContent = "About";
+h4Arr[2].textContent = "Services";
```

Comment:
This should all be brought down from the JSON file, not hard coded as they are here.